client/src/components/AppInfo.tsx

```diff
@@ -0,0 +1,50 @@
+import { BodyShort, Label } from '@navikt/ds-react'
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { HjelpemiddelItem } from '../interfaces/CommonTypes'
+
+type Props = {
+  hm: HjelpemiddelItem
+}
+
+export const AppInfo: React.FC<Props> = (props: Props) => {
+  const { t } = useTranslation()
+  const { hm } = props
+
+  if (!hm.appInfo) {
+    return (null)
+  }
```

Comment:
Dette er jo bare en preferanse fra min side, men jeg liker best å bruke conditional renders der componenten skal brukes, ala

```
{hm.appInfo &&
  <AppInfo hm={hm} />
}
```

Syns det er lettere å lese at den ikke alltid skal rendres, fremfor å måtte åpne komponenten for å se det.

Reply:
Jepp, kan vere enig det. Fix'd